### ISSUE TITLE
release 0.2.2

### ISSUE THREAD

**Issue by clarkliming** [insightsengineering/chevron#597]
- [x] #595
- [x] #592 
- [x] #601 (best included)
- [x] #605
- [x] prepare release
- [x] send it to autovalidator

Actions from release template:

Release date  -15-Aug

Pre-requisites
- [x] Discuss package dependencies before going into release activities.
- [x] Make sure all relevant integration tests are green 2-3 days before the release. Look carefully through logs (check for warnings and notes).
- [x] Inform about the soft code freeze, decide what gets merged in before starting release activities.

Release Checklist
- [x] Recurring tasks: Monitor integration tests, if integration fails, create priority issues on the board.
- [x] Update NEWS.md file: make sure it reflects a holistic summary of what has changed in the package, check README.
- [x] Remove the additional fields (`Remotes` and `Config/Needs/*`) from the DESCRIPTION file where applicable.
- [x] Make sure that the minimum dependency versions are updated in the DESCRIPTION file for the package.
  - [x] Increase versioned dependency on {package name} to >=X.X.X.
- [x] Create a pull request to make necessary bug fixes/changes (add "[skip vbump]" in the PR title), and after merging the PR, tag the update(s) as a release candidate v < intended release version > -rc < release candidate iteration > on the main branch. Note that tags are created in GitHub and synchronized with GitLab automatically.
- [x] The package is submitted for internal validation (Applicable only for regulatory release).
- [x] Address any feedback (internal validation/user testing), retag the package as a release candidate vX.X.X-rc(n+1). Repeat the submission for internal validation if necessary.
- [x] Get the package validated (Applicable only for regulatory release).
- [x] If the additional fields were removed, add them back in a separate PR, and then merge the PR back to main (add "[skip vbump]" in the PR title).
- [x] Create a git tag with the final version set to X.X.X on the main branch.
- [x] Update downstream package dependencies to (>=X.X.X) in {package name}.

Testing:
- [x] Integration tests results - accepted.
- [x] UAT results (if needed)- accepted.
- [x] Necessary testing on target environment - performed (up to ETL).

Post-release checklist:
- [x] Make sure that the package is published to internal repositories (Validated and/or Non-Validated repository).
~- [ ] Review and update installation instructions for the package if needed.~ N/A Installation remains the same
- [x] Verify if a new dev version (.9XXX) has been added to the NEWS.md file and DESCRIPTION file as a placeholder for release notes by automation.
- [x] Make sure internal documentation/documentation catalogs are up to date.
- [x] Notify the IDR team to start post-release/clean-up activities.
- [x] Announce the release on 21.08.
